### ISSUE TITLE
FaceID not available

### ISSUE THREAD

**Issue by MrBenFTW** [tihmstar/futurerestore#222]
After using future restore to upgrade from 11.1.2 to 11.2.1 I face a issue with FaceID not working .

**Comment by mtshare** [comment]
Please fix this!

**Comment by forum999** [comment]
agreed im dying to update but cant as tis happens to everyone on an iphone x using shsh blobs to update

**Comment by forum999** [comment]
seems no one is supporting this anymore?

**Comment by tihmstar** [comment]
there is no issue to fix here, this is know behavior. 
When using futurerestore you have to install a signed SEP, which usually is never than the firmware you are installing (downgrade iOS, but upgrade SEP). 
Sometimes if you are very lucky the SEP you are installing is very compatible with the iOS version (usually like 0.0.x updates). If you aren't as lucky but still luky then SEP is compatible enough to make your device boot (which is what happened in your case).
If you are unlucky, the SEP isn't compatible at all with the iOS you are trying to install in which case the device won't boot at all (bootloop and you have to restore to latest iOS). I personally wouldn't risk x.0.0 updates because of this unless someone else already confirmed it worked. 

So most of the cases (unless you happen to get a lucky iOS/SEP combination) it is expected that you can boot the device, but can't use TouchID/FaceID when restoring with futurerestore. 

I belive there should be a page on theiphonewiki where you can look up what iOS/SEP is know to work.

**Comment by MrBenFTW** [comment]
It is an issue, there’s a firmware called ‘savage’ which isn’t being parsed in the install. 

At the time of restoring which was ages ago, the SEP and Baseband was compatiable

**Comment by tihmstar** [comment]
if you know what's the issue is, why didn't you write it here right away?

**Comment by MrBenFTW** [comment]
I have in various other places, but this was 10 months ago. The iPhone X was basically just released and no one knew about savage
